.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 **Marked version:**
 
+**Markdown flavor:** Markdown.pl|CommonMark|GitHub Flavored Markdown|n/a
+
 <!-- The NPM version or commit hash having the issue --> 
```

Comment:
This comment is in the wrong place now. It should be after the **Marked Version** bold part.

Reply:
Updated - added change to initial PR comment.